lock-api-objects/src/main/java/com/palantir/lock/v2/PartialTimestampLeaseResult.java

```diff
@@ -20,7 +20,7 @@
 import org.immutables.value.Value;
 
 @Value.Immutable
-public interface AcquireNamedMinTimestampLeaseResult {
+public interface PartialTimestampLeaseResult {
     @Value.Parameter
     long minLeasedTimestamp();
```

Comment:
The list in this class should be a `TimestampRange`. I missed the refactor on the way. I will either modify here or add the change in a later PR in the stack.